recipes/neobolt/meta.yaml

```
@@ -0,0 +1,48 @@
+{% set name = "neobolt" %}
+{% set version = "1.7.4" %}
+{% set sha256 = "3324f2b319e84acb82e37a81ef75f3f7ce71c149387daf900589377db48bed2a" %}
+
+package:
+  name: {{ name|lower }}
+  version: {{ version }}
+
+source:
+  url: https://pypi.io/packages/source/{{ name[0] }}/{{ name }}/{{ name }}-{{ version }}.tar.gz
+  sha256: {{ sha256 }}
+
+build:
+  number: 0
+  script: python -m pip install . --no-deps -vv
+
+requirements:
+  build:
+    - {{ compiler('c') }}
+  host:
+    - python
+    - pip
+  run:
+    - python
+    - pytz
+    - neotime
+
+test:
+  imports:
+    - neobolt
+    - neobolt.direct
+    - neobolt.exceptions
+    - neobolt.routing
+
+about:
+  home: http://neobolt.readthedocs.io
+  license: Apache-2.0
+  license_family: Apache
+  license_file: '{{ environ["RECIPE_DIR"] }}/LICENSE'
```

There is no need to explicitly state the recipe directory, conda will automatically look there if it isn't found in the build directory:
```suggestion
  license_file: LICENSE
```